### ISSUE TITLE
Marking a person's attendance could be improved

### ISSUE THREAD

**Issue by tyouwei** [tyouwei/pe#6]
Say for example the TA wants to mark the 2nd person's attendance and the first person has the exact same name as the 2nd, the app will mistakenly mark the attendance for the 1st card instead.

I understand that the ID is unique and can be used. But having the feature to mark by name could result in mistakes in attendance marking, especially when the class is huge and difficult to eyeball if two entries are the same. 

tldr; better to remove the feature to mark by name

![image.png](https://raw.githubusercontent.com/tyouwei/pe/main/files/b2c24cd3-9fa9-416e-b0ae-d24aa228aced.png)
